docs/ja-jp/3.6-インフラスケジュール設定.md

```diff
@@ -0,0 +1,108 @@
+# インフラスケジュール設定
+
+`resource/custom_infrast/*.json` 各フィールドの設定方法と説明
+
+## 完整字段一览
+
+```jsonc
+{
+    "title": "小号的换班方案",       // タイトル，オプション
+    "description": "哈哈哈哈",       // 説明，オプション
+    "plans": [
+        {
+            "name": "早班",         // プランタイトル，オプション
+            "description": "lol",   // 説明，オプション
+            "period": [             // シフト時間の間隔，オプション
+                                    // 現在時刻がこの間隔内にある場合、プランが自動的に選択されます (jsonファイル全体が複数のプランを含む場合もあります)
+                                    // core ではこのフィールドを扱わないので、maa を統合するためにインターフェイスを使用する場合は、このロジックを自分で実装してください。
+                [
+                    "22:00",        // 必須　記述のフォーマットは hh:mm， 日をまたぐ場合はこの例のように記述してください。
+                    "23:59"
+                ],
+                [
+                    "00:00",
+                    "06:00"
+                ]
+            ],
+            "duration": 360,        // 作業時間(分) 予約項目, 未実装 将来的には、シフトを変更時間を告知させるポップアップ時間になるかも？ 自動的に変更される機能になる可能性もあり
+            "Fiammetta": {          // “フィアメッタ” どのオペレーターを配属するか，任意，記入しない場合は配属しない。
+                "target": "シャマレ",   // ターゲットオペレーター この記述例の場合フィアメッタとシャマレの体力交換スキーム
```

Comment:
I forgot to ask the question.
As for the target field, in OCR it is client language dependent, but which is the notation standard here?

Reply:
Maybe use this is OK.
What's the other possible notation standard?

Reply:
下の方のOCR使用部分はクライアント依存であったため、こちらの部分はOCR使用なのか内部データ使用なのか判断ができないので質問させていただきました。
クライアント依存のOCR使用で問題ないのであればこのままPushしていただいて大丈夫だと思います。

I asked this question because I am unable to determine whether this part uses OCR or internal data, since the part below that uses OCR is client-dependent.
If there is no problem with client-dependent OCR use, I think it is fine for you to push as is.

Reply:
谢谢啦~
不用担心OCR这部分的内容，这里的东西在之后OCR代码更改的时候会由我们相应修改的。现在直接写原名就是最好的写法。

Thank you~
Don't worry about the content of the OCR part, the things here will be modified by us when the OCR code changes later. Now writing the original name directly is the best way to write it.